rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json

```diff
@@ -1105,6 +1105,53 @@
         "minLength": 4
       }
     },
+    "editionStatement": {
+      "title": "Edition Statement",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "editionDesignation": {
+            "title": "Edition Designation",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "string",
+                  "minLength": 1
+                },
+                "language": {
+                  "title": "Language",
+                  "type": "string",
+                  "$ref": "#/definitions/language_script"
+                }
+              }
+            }
+          },
+          "responsibility": {
+            "title": "Responsibility",
+            "type": "array",
```

Comment:
minItems

Reply:
no min array, responsibility is not mandatory